fonts/Makefile.am

```diff
@@ -27,13 +27,15 @@ TTF_GREGORIO_OP = gregorio-op.ttf gregorio-op-hollow.ttf gregorio-op-hole.ttf
 TTF_GRANAPADANO = granapadano.ttf granapadano-hollow.ttf granapadano-hole.ttf
 TTF_GRANAPADANO_OP = granapadano-op.ttf granapadano-op-hollow.ttf granapadano-op-hole.ttf
 TTF_EXTRA = greextra.ttf gregall.ttf gresgmodern.ttf grelaon.ttf
-TTF_ALL = $(TTF_GRECILIAE) $(TTF_GRECILIAE_OP) $(TTF_GREGORIO) \
-		  $(TTF_GREGORIO_OP) $(TTF_GRANAPADANO) $(TTF_GRANAPADANO_OP)
-SOURCEFILES = gregorio-base.sfd granapadano-base.sfd greciliae-base.sfd \
-			  greextra.sfd gregall.sfd gresgmodern.sfd grelaon.sfd
+TTF_ALL = $(TTF_GRECILIAE) $(TTF_GRECILIAE_OP) $(TTF_GREGORIO) $(TTF_GREGORIO_OP) $(TTF_GRANAPADANO) $(TTF_GRANAPADANO_OP)
```

Comment:
Ascetically I don't like them either, but I had some trouble with tabs vs. spaces in my editor and forcing them to one line was the simplest way to get around the issue.  I'll see if I can't solve that issue and get rid of the long lines.